sympy/geometry/ellipse.py

```diff
@@ -402,8 +439,30 @@ def encloses_point(self, p):
 
         return fuzzy_bool(test.is_positive)
 
-    def equation(self, x='x', y='y'):
-        """The equation of the ellipse.
+    def equation(self, x='x', y='y', slope=None):
+        """
+        Returns the equation of an ellipse either using the slope of major axis, or, using hradius
+        and vradius.
+
+        Approach to the solution (using slope) :
+
+        Let the center of the ellipse be at C=(xc,yc)
+
+        Let the major axis be the line that passes through C with a slope of s;
+        points on that line are given by the zeros of L(x,y) = (y−yc − s*(x−xc))**2.
```

Comment:
The docstring contains unicode minus signs.

Reply:
@sidhantnagpal  Yes. You're right. I missed that. Corrected now. Please take a look.